### DISCUSSION TITLE
Terminated protein is genrated

### DISCUSSION THREAD

**Discussion by H440775h** [facebookresearch/esm#325] [ESM Metagenomic Atlas]
I tried with differt protein sequences, but the genrated structure is only for partial sequnces not for full protein sequences

**Comment by claudiostrabio** [comment]
Had exactly the same problem
Sequence entered: 
MRAQEDLEGRTQHETTRDPSTPLPTEPKFDMLYKIEDVPPWYLCILLGFQHYLTCFSGTIAVPFLLAEAL
CVGHDQHMVSQLIGTIFTCVGITTLIQTTVGIRLPLFQASAFAFLVPAKAILALERWKCPPEEEIYGNWS
LPLNTSHIWHPRIRDVQGAIMVSSVVEVVIGLLGLPGALLNYIGPLTVTPTVSLIGLSVFQAAGDRAGSH
WGISACSILLIILFSQYLRNLTFLLPVYRWGKGLTLLRIQIFKMFPIMLAIMTVWLLCYVLTLTEVLPTD
PKAYGFQARTDARGDIMAIAPWIRIPYPCQWGLPTVTAAAVLGMFSATLAGIIESIGDYYACARLAGAPP
PPVHAINRGIFTEGICCIIAGLLGTGNGSTSSSPNIGVLGITKVGSRRVVQYGAAIMLVLGTIGKFTALF
ASLPDPILGGMFCTLFGMITAVGLSNLQFVDMNSSRNLFVLGFSMFFGLTLPNYLSPNPGAINTGILEVD
QILIVLLTTEMFVGGCLAFILDNTVPGSPEERGLIQWKAGAHANSDMSSSLKSYDFPIGMGIVKRITFLK
YIPICPVFKGFSSSSKDQIAIPEDTPENTETASVCTKV

Sequence used:
MRAQEDLEGRTQHETTRDPSTPLPTEPKFDMLYKIEDVPPWYLCILLGFQHYLTCFSGTIAVPFLLAEALCVGHDQHMVSQLIGTIFTCVGITTLIQTTVGIRLPLFQASAFAFLVPAKAILALERWKCPPEEEIYGNWSLPLNTSHIWHPRIRDVQGAIMVSSVVEVVIGLLGLPGALLNYIGPLTVTPTVSLIGLSVFQAAGDRAGSH

**Comment by tomsercu** [comment]
Does this issue occur with the web interface, the API, or when running the models yourself?
